charter/index.html

```diff
@@ -318,19 +321,11 @@ <h3>
   Richard Ishida, Fuqiao Xue
 </address>
 
-<p class="copyright">
-  <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> ©
-  2021
-  <a href="https://www.w3.org/"><abbr title="World Wide Web Consortium">W3C</abbr></a><sup>®</sup>
-  (
-  <a href="https://www.csail.mit.edu/"><abbr title="Massachusetts Institute of Technology">MIT</abbr></a>,
-  <a href="https://www.ercim.eu/"><abbr title="European Research Consortium for Informatics and Mathematics">ERCIM</abbr></a>,
-  <a href="https://www.keio.ac.jp/">Keio</a>,
-  <a href="https://ev.buaa.edu.cn/">Beihang</a>
-  ), All Rights Reserved.
-
-  <abbr title="World Wide Web Consortium">W3C</abbr> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a href="https://www.w3.org/Consortium/Legal/copyright-documents">document use</a> rules apply.
-</p>
+<p class="copyright">Copyright © 2023 <a href="https://www.w3.org/">World Wide Web Consortium</a>.<br>
```

Comment:
Is it possible to get the current year automatically using JS here?

Reply:
Thanks for your suggestion. We treat charters as static documents/snapshots, and the year is only updated when the charter is updated, so I don't recommend using a script to automatically update it.